name.abuchen.portfolio.tests/src/name/abuchen/portfolio/datatransfer/pdf/degiro/DegiroPDFExtractorTest.java

```diff
@@ -8758,4 +8804,64 @@ public void testTransacoes01()
         assertThat(entry.getPortfolioTransaction().getUnitSum(Unit.Type.FEE),
                         is(Money.of(CurrencyUnit.EUR, Values.Amount.factorize(0.00))));
     }
+    
+    /**
+     * Utility function to check one security 
+     * 
+     * @param security
+     * @param expName
+     * @param expIsin
+     * @param expCurrency
+     */
+    private void checkOneSecurity( Security security, String expName, String expIsin, String expCurrency) 
+    { 
+        assertThat(security.getName(), is(expName));
+        assertThat(security.getIsin(), is(expIsin));
+        assertThat(security.getCurrencyCode(), is(expCurrency));
+    }
+    
+    /**
+     * Utility to check one Buy/Sell transaction
+     *  
+     * @param entry
+     * @param portfolioTransactionType
+     * @param accountTransactionType
+     * @param dateTime
+     * @param shares
+     * @param currency
+     * @param monetaryAmount
+     * @param grossValue
+     * @param taxSum
+     * @param feeSum
+     * @param forexCurrency
+     * @param forexValue
+     */
+    private void checkOneBuySellEntry( BuySellEntry entry, PortfolioTransaction.Type portfolioTransactionType, AccountTransaction.Type accountTransactionType, 
+                    String dateTime, double shares, String currency, double monetaryAmount, 
+                    double grossValue, double taxSum, double feeSum, String forexCurrency, double forexValue)
+    {
+        
+        assertThat(entry.getPortfolioTransaction().getType(), is(portfolioTransactionType));
+        assertThat(entry.getAccountTransaction().getType(), is(accountTransactionType));
+
+        assertThat(entry.getPortfolioTransaction().getDateTime(), is(LocalDateTime.parse(dateTime)));
+        assertThat(entry.getPortfolioTransaction().getShares(), is(Values.Share.factorize(shares)));
+
+        assertThat(entry.getPortfolioTransaction().getMonetaryAmount(),
+                        is(Money.of(currency, Values.Amount.factorize(monetaryAmount))));
+        assertThat(entry.getPortfolioTransaction().getGrossValue(),
+                        is(Money.of(currency, Values.Amount.factorize(grossValue))));
+        assertThat(entry.getPortfolioTransaction().getUnitSum(Unit.Type.TAX),
+                        is(Money.of(currency, Values.Amount.factorize(taxSum))));
+        assertThat(entry.getPortfolioTransaction().getUnitSum(Unit.Type.FEE),
+                        is(Money.of(currency, Values.Amount.factorize(feeSum))));
+        
+        if (forexCurrency != null)
+        {
+            Unit grossValueUnit = entry.getPortfolioTransaction().getUnit(Unit.Type.GROSS_VALUE)
+                            .orElseThrow(IllegalArgumentException::new);
+            assertThat(grossValueUnit.getForex(), is(Money.of(forexCurrency, Values.Amount.factorize(forexValue))));
+        }
+
+    }
```

Comment:
Hello @couclock 
please undo this. We already have a simple test variant.

As you can see in the other tests and also read in the [contributing rules](https://github.com/portfolio-performance/portfolio/blob/master/CONTRIBUTING.md#test-cases-1), we use these variants.

From April 2023 we will use the simplified notation of test cases (preferred variant)
[ComDirect Bank](https://github.com/portfolio-performance/portfolio/blob/master/name.abuchen.portfolio.tests/src/name/abuchen/portfolio/datatransfer/pdf/comdirect/ComdirectPDFExtractorTest.java)
        [Baader Bank](https://github.com/portfolio-performance/portfolio/blob/master/name.abuchen.portfolio.tests/src/name/abuchen/portfolio/datatransfer/pdf/baaderbank/BaaderBankPDFExtractorTest.java) with testWertpapierKauf23()
        [Sbroker](https://github.com/portfolio-performance/portfolio/blob/master/name.abuchen.portfolio.tests/src/name/abuchen/portfolio/datatransfer/pdf/sbroker/SBrokerPDFExtractorTest.java) with testDividende11()
        [Sbroker](https://github.com/portfolio-performance/portfolio/blob/master/name.abuchen.portfolio.tests/src/name/abuchen/portfolio/datatransfer/pdf/sbroker/SBrokerPDFExtractorTest.java) with testGiroKontoauszug10()